asv.conf.json

```diff
@@ -0,0 +1,160 @@
+{
```

Comment:
Why is everything red?

Reply:
Because json doesn't support comments by default.